cpp/src/sampling/neighbor_sampling_impl.hpp

```diff
@@ -65,8 +69,10 @@ uniform_neighbor_sample_impl(
 {
 #ifdef NO_CUGRAPH_OPS
```

Comment:
Is this check still correct?

I believe cugraph-ops is only required if edge_bias_view is `std::nullopt`.   If edge_bias_view is specified then we're not using cugraph-ops.

Of course, removing this ifdef and checking at runtime may require creating an #ifdef in the sampling primitive somewhere to make sure that we don't try and include the cugraph-ops code.  Haven't checked to see what the plan is.

Perhaps a FIXME would be sufficient, since we hope to break the dependency on cugraph-ops in the release anyway.

Reply:
Yes, this `ifdef` is relevant only when `edge_bias_view` is std::nullopt and we can defer this check till sampling primitives.

My thought was that we will fix this anyways when we migrate cugraph-ops functions to cugraph, so better fix this at once then creating an intermediate fix.

I can add FIXME as suggestsed.